examples/tutorials/next.md

```diff
@@ -269,20 +295,25 @@ return (
 );
 ```
 
+### Add some styles
+
+Let's add some basic styles to make the app look nicer. Update your
+`app/globals.css` file with the
+[styles from this file](https://raw.githubusercontent.com/denoland/tutorial-with-next/refs/heads/main/src/app/globals.css).
+
 ## Run the app
 
-Now you can run the app with `deno task dev` and visit `http://localhost:3000`
-in your browser to see the list of dinosaurs. Click on a dinosaur to see more
+Now you can run the app with `deno run dev` and visit `http://localhost:3000` in
+your browser to see the list of dinosaurs. Click on a dinosaur to see more
 details!
 
-![demo of the app](./images/how-to/next/dinoapp.gif)
+## Deploy the app
 
-## Deploy your app
+Now that you have your working Next.js app, you can deploy it to the web with
+Deno Deploy<sup>EA</sup>.
```

Comment:
I'd keep these git instructions, but frame them as "or you can do this manually by..." after offering first a Deploy button. 

Even though the image for the button is still pending, we can provide it as a link. 

`https://app.deno.com/new?clone=REPO_URL`

I suggest some sort of "Deploy a copy of this project by clicking this link/button" language, and then going on to say what the button does, and how you can do that manually.